<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AverageWindow</class>
 <widget class="QDockWidget" name="AverageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="allowedAreas">
   <set>Qt::BottomDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Average manager</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QLabel" name="m_label_EvokedSets">
      <property name="text">
       <string>Evoked sets:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>2D Layout</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QPushButton" name="m_pushButton_exportLayoutPlot">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>85</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Export plot</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QGraphicsView" name="m_graphicsView_layout"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Butterfly graph</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="2">
           <widget class="QPushButton" name="m_pushButton_exportButterflyPlot">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Export Plot</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="m_comboBox_channelKind">
            <property name="minimumSize">
             <size>
              <width>84</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>84</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>MEG_grad</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MEG_mag</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EEG</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QGraphicsView" name="m_graphicsView_butterflyPlot"/>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="m_tableView_loadedSets">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Plots:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
